core/src/main/java/org/apache/accumulo/core/util/tables/TableZooHelper.java

```diff
@@ -59,6 +62,12 @@ public TableZooHelper(ClientContext context) {
    *         getCause() of NamespaceNotFoundException
    */
   public TableId getTableId(String tableName) throws TableNotFoundException {
+    if (MetadataTable.NAME.equals(tableName)) {
```

Comment:
In elasticity we could add a `static Map<String, TableId> getAllNames()` method to AccumuloTable similar to [this existing method](https://github.com/apache/accumulo/blob/b2caa1f07a04cb18e27d376d7d1ce7054b4c6fce/core/src/main/java/org/apache/accumulo/core/metadata/AccumuloTable.java#L57).  That map could be used for this check and it would cover all the new tables in the accumulo namespace in elasticity.

Reply:
Yeah, good point. I knew this was going to have to change on the merge.